### DISCUSSION TITLE
Just make the cards little bigger

### DISCUSSION THREAD

**Discussion by IamNickerson** [community/community#13291] [Other Feature Feedback, Questions, & Ideas]
The card size should be bigger because if a long text comes it could break the card so it is much better to make it bigger not like big big just make it little big.
![image](https://user-images.githubusercontent.com/61643704/159603682-ed36b8d0-60e9-495e-beef-9621c4fda00d.png)

**Comment by sntwandy** [comment]
I think because of the habit of the other feed where the cards were bigger maybe, but it would be a good idea to test how it would look a little bigger and if it doesn't affect the design system.

**Reply by IamNickerson** [reply]
Okay.
